package/gluon-mesh-vpn-wireguard/src/gluon-mesh-vpn-key-translate.c

```diff
@@ -0,0 +1,82 @@
+// SPDX-FileCopyrightText: 2022 Jan-Niklas Burfeind <gluon@aiyionpri.me>
+// SPDX-License-Identifier: BSD-2-Clause
+// SPDX-FileContributor: read_key() by Matthias Schiffer <mschiffer@universe-factory.net>
+
+#include <libubox/utils.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#define FASTD_KEY_LEN_HEX 64
+
+static inline void invalid_key(void) {
+	fprintf(stderr, "Error: Invalid fastd key provided.\n");
+	exit(1);
+}
+
+static inline void no_mem(void) {
+	fprintf(stderr, "Error: Not enough memory for base64 encoding available.\n");
+	exit(1);
+}
+
+static inline void usage(void) {
+	fprintf(stderr, "Usage: gluon-mesh-vpn-key-translate [-n] <fastd secret>\n");
+	fprintf(stderr, "-n     do not output the trailing newline\n");
+	exit(1);
+}
+
+static inline bool read_key(uint8_t key[32], const char *hexkey) {
+	if ((strlen(hexkey) != 64) || (strspn(hexkey, "0123456789abcdefABCDEF") != 64))
```

Comment:
The braces are superfluous and don't contribute to code readability.

Reply:
You are right, there are braces one could've left out regarding precedence;
but I'd disagree on the readability. Both does not really matter though, what does is, that that's not my code, but actually @NeoRaider's, as attributed above.
https://github.com/NeoRaider/fastd/blob/d9dd14045c647736566c959855f154302eeff444/src/protocols/ec25519_fhmqvc/ec25519_fhmqvc.c#L17-L27

But as I said earlier I'd really like to get some basic feedback from neoraider, before I address nitpicks.

Statistically speaking - looking at how my previous PR's went -there will likely be no line left unturned; really those braces will be the least of concerns, once he finds time, I'm afraid.

Not that I do not appreciate the engagement with the PR,
but beautifying is not the stage I'm currently at, yet,